avm/src/vm/deploy.rs

```diff
@@ -0,0 +1,185 @@
+use hyper::{client::Client, Body, Request};
+use serde::{Deserialize, Serialize};
+use serde_json::{from_reader, from_str, json, Value};
+
+use std::error::Error;
+use std::fmt::Display;
+use std::fs::{File, read};
+use std::io::BufReader;
+use std::path::Path;
+
+use ascii_table::{AsciiTable, Column, Align};
+use base64;
+use tokio::runtime::Runtime;
+
+const URL: &str = "http://deploy.alantechnologies.com";
```

Comment:
Next comment; we need to make sure this is https if we're transmitting AWS credentials through this. Getting a cert shouldn't be too hard with Let's Encrypt or whatever Heroku has built-in.

Reply:
Oh it works with https. That's a typo. Will update tomorrow.